### PR TITLE
Manipulate surface WP

### PR DESCRIPTION
**Pull request by aslakbergersen**
Add a separate script for controlling the surface roughness of a model. The method can both smooth and add noise to a surface. The noise is either added through adding new points to the Voronoi diagram or by perturbing the existing points.

The smoothing algorithm is the same as is included in the other methods but includes the alternative of an absolute tolerance for the smoothing.

**Work left**
- Add a demo file
- Add a test
- Add a tutorial page

### CODE REVIEW

File: morphman/manipulate_area.py [hunk 1]
```diff
@@ -274,15 +274,15 @@ def change_area(voronoi, factor, line_to_change, diverging_centerline, diverging
...
 
         v_change = D - B
         if sign > 0:
-            factor_ = factor[tmp_id1] * (1 - np.linalg.norm(A-D) / AC_length) \
-                    + factor[tmp_id2] * (np.linalg.norm(A-D) / AC_length)
+            factor_ = factor[tmp_id1] * (1 - np.linalg.norm(A - D) / AC_length) \
+                      + factor[tmp_id2] * (np.linalg.norm(A - D) / AC_length)
             if factor_ > 10:
                 print(factor_, factor[tmp_id1], factor[tmp_id2], AC_length,
                       np.linalg.norm(c_change))
```
Comment (hkjeldsberg) [inline morphman/manipulate_area.py]: Kan ikke se at `c_change` er definert. Skulle det vært:
```suggestion
                      np.linalg.norm(v_change))
```

Comment (aslakbergersen) [reply morphman/manipulate_area.py]: Det stemmer nok, men mer generelt så burde hele den if-testen tas bort. Jeg antar at du har gjort det i asymmetric branchen?

Comment (hkjeldsberg) [reply morphman/manipulate_area.py]: Ja, skulle til å spørre hva denne snutten gjorde. Det divergerer heftig godt med bulge / stenosis, så gjør en sjekk på metode før if testen i asymmetric-branchen. Men kan godt fjerne hele greia.

File: morphman/common/vmtk_wrapper.py [hunk 2]
```diff
@@ -260,3 +260,48 @@ def vmtk_endpoint_extractor(centerlines, clipspheres):
...
+    curvature.surface
+    curvature.CurvatureType = curvature_type
+    if absolute:
+        curvature.AbsoluteCurvature = 1
+    else:
+        curvature.AbsoluteCurvature = 0
+    if median:
+        curvature.MedianFiltering = 1
+    else:
+        curvature.medianFiltering = 0
```
Comment (hkjeldsberg) [inline morphman/common/vmtk_wrapper.py]:
```suggestion
        curvature.MedianFiltering = 0
```

**Pull request merged**